style.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,300;0,400;1,300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;1,100;1,300;1,400;1,500;1,700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+
+  /* overflow: hidden; */
+}
+
+.toolbar_container {
+  display: flex;
+  flex-direction: row;
+  background-color: #3c3a39;
+  max-width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  align-content: center;
+  justify-items: center;
+  gap: 180px;
+  height: 17vh;
+  padding-bottom: 5rem;
+}
```

Comment:
- [ ] As highlighted by the previous reviewers, you need to redesign the `header` section to look identical to the header `section` of the Figma template. There is too much space around the elements in the `header` section. 👍 😎 

### Your Design

![image](https://github.com/malodev1234/Portfolio_setup_and_mobile_First/assets/77106962/9b10db44-2b1e-4127-a083-8a97eb50a14d)

### Figma Design

![image](https://github.com/malodev1234/Portfolio_setup_and_mobile_First/assets/77106962/fa94a412-cda0-4f70-aa23-ef06cfa2f01d)

#### Hint: Please ignore the time, network and battery icons. They depict how the design looks on an actual mobile phone. It would be best if you concentrated on the logo and hamburger icon only. Therefore, I would recommend you remove these declarations in the `.toolbar_container` selector 💻:

```css
height: 17vh;
padding-bottom: 5rem;
```
#### Then add this declaration

```css
padding: 1rem 1.5rem;
```